src/map/map.ts

```diff
@@ -210,13 +210,13 @@ export class PropertiesMap {
      * Create a new [[PropertiesMap]] inside the DOM element with the given HTML
      * `id`
      *
-     * @param id         HTML id of the DOM element where the map should live
+     * @param element   HTML element or HTML id of the DOM element where the name should live
```

Comment:
copy-paste error (name => map) and same comment as above ("HTMLElement or string id ...")